setup/building-es2k-stratum-deps.md

```diff
@@ -0,0 +1,221 @@
+# Building Stratum Dependencies for the ES2K ACC
+
+This document explains how to build the Stratum dependencies for the
+ARM Compute Complex (ACC) of the Intel&reg; IPU E2100.
+
+> **Note**: To build the dependencies for a different target, see
+[Building the Stratum Dependencies](building-stratum-deps.md).
+
+## 1. Introduction
+
+Stratum is the component of P4 Control Plane that implements the P4Runtime
+and gNMI (OpenConfig) services. It requires a number of third-party
+libraries, which this package provides.
+
+You will need to build two versions of the libraries:
+
+- The **Host** libraries, which run on the build system. They provide
+  tools that are used to build P4 Control Plane.
+
+- The **Target** libraries, which run on the ACC. P4 Control Plane is
+  compiled and linked against these libraries.
+
+The Host and Target libraries must be the same version.
+
+## 2. Preparing the System
+
+There are several things to do before you can build the dependencies.
+
+- Install CMake 3.15 or above
+
+  Avoid versions 3.24 and 3.25. They cause the dependencies build to fail.
+
+- Install OpenSSL 1.1
```

Comment:
We have OpenSSL version 3.0 in some versions of Linux such as Fedora 37, Rocky Linux 9.1.

Reply:
To the best of my knowledge, we have only ever tested against OpenSSL 1.1.

You and I discussed this a couple of months ago. Did you every get a definitive answer regarding building and testing against, and compatibility with, OpenSSL 3.0?

Also, as I recall, there was also an argument that we should move to 3.1.1. I don't remember what it was: possibly an IPAS recommendation, or the availability of something FIPS-related?

In any case, I'd like some assurance before saying anything about 3.0 in the documentation.

Reply:
yes, I remember that discussion.
I believe most of the testing has been performed on Fedora 37 and Rocky Linux 9.1 which has OpenSSL 3.0 by default. 

Let's re-confirm before adding anything to documentation. I am ok with this for now for initial doc.